conmon/conmon.c

```diff
@@ -745,6 +749,47 @@ int main(int argc, char *argv[])
 	cpid = atoi(contents);
 	ninfo("container PID: %d", cpid);
 
+	/* Setup endpoint for attach */
+	char attach_symlink_dir_path[PATH_MAX];
+	struct sockaddr_un attach_addr = {0};
+	_cleanup_close_ int afd = -1;
+
+	if (!exec) {
+		attach_addr.sun_family = AF_UNIX;
+
+		/*
+		 * Create a symlink so we don't exceed unix domain socket
+		 * path length limit.
+		 */
+		snprintf(attach_symlink_dir_path, PATH_MAX, "/run/%s", cid);
```

Comment:
Its somewhat weird to me to use /run here, where everywhere else we use /var/run. I mean, typically /var/run is a symlink to /run, but still. We should probably be consistent anyway.